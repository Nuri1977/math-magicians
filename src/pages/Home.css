.home-title {
  padding: 2rem 3rem;
}

.home-pg {
  padding: 1rem 3rem;
}
