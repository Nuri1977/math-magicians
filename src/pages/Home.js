import React from 'react';
import './Home.css';

export default function Home() {
  return (
    <div className="container">
      <h1 className="home-title">Welcome to our page!</h1>
      <p className="home-pg">
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Libero voluptatibus, recusandae, temporibus dignissimos
        ullam porro earum repellendus explicabo
        esse officiis ratione nulla vero
        aliquam nisi molestias placeat, duci
        mus quos nobis.
      </p>
      <p className="home-pg">
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Libero voluptatibus, recusandae, temporibus dignissimos
        ullam porro earum repellendus explicabo
        esse officiis ratione nulla vero
        aliquam nisi molestias placeat, duci
        mus quos nobis.
      </p>
    </div>
  );
}
