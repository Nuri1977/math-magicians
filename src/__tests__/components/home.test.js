import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

describe('Testing home component', () => {
  it('Home renders correctly', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
