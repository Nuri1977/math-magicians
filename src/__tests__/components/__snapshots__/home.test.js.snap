// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing home component Home renders correctly 1`] = `
<div
  className="container"
>
  <h1
    className="home-title"
  >
    Welcome to our page!
  </h1>
  <p
    className="home-pg"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero voluptatibus, recusandae, temporibus dignissimos ullam porro earum repellendus explicabo esse officiis ratione nulla vero aliquam nisi molestias placeat, duci mus quos nobis.
  </p>
  <p
    className="home-pg"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero voluptatibus, recusandae, temporibus dignissimos ullam porro earum repellendus explicabo esse officiis ratione nulla vero aliquam nisi molestias placeat, duci mus quos nobis.
  </p>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <h1
    className="home-title"
  >
    Welcome to our page!
  </h1>
  <p
    className="home-pg"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero voluptatibus, recusandae, temporibus dignissimos ullam porro earum repellendus explicabo esse officiis ratione nulla vero aliquam nisi molestias placeat, duci mus quos nobis.
  </p>
  <p
    className="home-pg"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Libero voluptatibus, recusandae, temporibus dignissimos ullam porro earum repellendus explicabo esse officiis ratione nulla vero aliquam nisi molestias placeat, duci mus quos nobis.
  </p>
</div>
`;
